src/pipecat/services/ai_services.py

```diff
@@ -46,6 +47,11 @@
 class AIService(FrameProcessor):
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
+        self.model_name: str = ""
```

Comment:
could we make this private and a property instead?

```
@property
def model_name(self):
   return self._model_name
```

Reply:
done. i think this is ready for final eyes :)